spec/models/keyword_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Keyword, type: :model do
+  describe 'Init' do
+    let(:keyword) { Fabricate(:keyword) }
```

Comment:
```suggestion
```

Same with the comment here: https://github.com/blyscuit/rails-ic/pull/32/files#r1217354441

Reply:
Thank you very much.